main.js

```diff
@@ -563,14 +575,28 @@ app.on('ready', function() {
 
       require('open')('mailto:project@openbazaar.org?subject=OpenBazaar Debug Report&body=' + body);
 
-    }},
-    {type: 'separator'},
+    }}
+  ];
+
+  if(launched_from_installer) {
+    template.push({label: 'View Python Error Log', type: 'normal', click: function() {
+      var logPath = __dirname + path.sep + 'python_error.log';
+      require('open')(logPath);
```

Comment:
Since multiple places are bringing in the "open" module, can we bring it in up top and assign it to a variable. Then, just use the variable?

```
var open = require('open');
```

then change the multiple:

```
require('open')(blah);
```

to

```
open('blah');
```